src/callbacks_step/stepsize_dg1d.jl

```diff
@@ -18,30 +19,71 @@ function max_dt(u, t, mesh::TreeMesh{1},
             lambda1, = max_abs_speeds(u_node, equations)
             max_lambda1 = max(max_lambda1, lambda1)
         end
-        inv_jacobian = cache.elements.inverse_jacobian[element]
+        inv_jacobian = cache.elements.inverse_jacobian[element] # Δx
```

Comment:
```suggestion
        inv_jacobian = cache.elements.inverse_jacobian[element] # 2 / Δx
```
On a 1D mesh the Jacobian is dx/2 so the inverse quantity would be 2 / dx